models/intermediate/int_pendo__feature_info.sql

```diff
@@ -0,0 +1,62 @@
+with feature as (
+
+    select 
+        {{ dbt_utils.star(from=ref('int_pendo__latest_feature'), except='LATEST_FEATURE_INDEX' if target.type == 'snowflake' else 'latest_feature_index') }}
```

Comment:
Can you do a quick confirmation check here that the substring for `feature` will not be removed in Snowflake? This does not happen in BQ, but my work on SFDC Formulas make me a bit skeptical.

Reply:
which column are you thinking of? i don't think there are any that are substrings of `latest_feature_index`